exchanges/multipart-fetch/src/multipartFetchExchange.test.ts

```diff
@@ -0,0 +1,212 @@
+import { Client, OperationResult, OperationType } from '@urql/core';
+import { empty, fromValue, pipe, Source, subscribe, toPromise } from 'wonka';
+import gql from 'graphql-tag';
+import { print } from 'graphql';
+
+import { multipartFetchExchange, convertToGet } from './multipartFetchExchange';
+import { uploadOperation, queryOperation } from './utils';
+
+const fetch = (global as any).fetch as jest.Mock;
+const abort = jest.fn();
+
+const abortError = new Error();
+abortError.name = 'AbortError';
+
+beforeAll(() => {
+  (global as any).AbortController = function AbortController() {
+    this.signal = undefined;
+    this.abort = abort;
+  };
+});
+
+afterEach(() => {
+  fetch.mockClear();
+  abort.mockClear();
+});
+
+afterAll(() => {
+  (global as any).AbortController = undefined;
+});
+
+const response = {
+  status: 200,
+  data: {
+    data: {
+      user: 1200,
+    },
+  },
+};
+
+const exchangeArgs = {
+  forward: () => empty as Source<OperationResult>,
+  client: {} as Client,
+};
+
+describe('on success', () => {
+  beforeEach(() => {
+    fetch.mockResolvedValue({
+      status: 200,
+      json: jest.fn().mockResolvedValue(response),
+    });
+  });
+
+  it('uses a file when given', async () => {
```

Comment:
Should we add a test with multiple files?

Reply:
Good shout